neurokit/signal/complexity.py

```diff
@@ -405,17 +410,37 @@ def complexity_entropy_multiscale(signal, emb_dim=2, tolerance="default"):
         # Replaced the sample entropy computation with nolds' one...
 #        se = sample_entropy(temp_ts, 1, tolerance)
 
-        try:
-            se = nolds.sampen(temp_ts, 1, tolerance, dist="euclidean", debug_plot=False, plot_file=None)
-        except:
-            se = nolds.sampen(temp_ts, 1, tolerance, dist="euler", debug_plot=False, plot_file=None)
+        # Replaced with nolds command for dist - string inputs are deprecated. 
+#        try:
+#            se = nolds.sampen(temp_ts, emb_dim, tolerance, dist="euclidean", debug_plot=False, plot_file=None)
+#        except:
+#            se = nolds.sampen(temp_ts, emb_dim, tolerance, dist="euler", debug_plot=False, plot_file=None)
+        se = nolds.sampen(temp_ts, emb_dim, tolerance, nolds.measures.rowwise_euclidean, debug_plot=False, plot_file=None)
 
 
         mse[0, i] = se
 
-    multiscale_entropy = mse[0][emb_dim-1]
+    multiscale_entropy = mse[0][max_scale_factor-1]
+    """
+    Unless I am missing something, I think there is an error with the calc of multiscale_entropy. 
+    MSE entropy from the original costa paper (refed in this fx)shows comparisons of the entropy 
+    for each scale factor. Recent implementations have condensed this into a single measurement. 
+    This single measurement seems to typically be the area under a curve, where the curve is 
+    created by the points from the individual scale factors. In contrast, the current 
+    implementaiton seems to just take the last value in the array - so only taking a single time 
+    scale measurement, and the last one. I think that if this is the desired output, it could simply 
+    be done by forgoing the entire for loop and replacing i with the scale_factor. 
+    
+    Here is a potential method to get the area: 
+
+    multiscale_entropy= numpy.trapz(multiscale_entropy)
+    """
+
+    if return_coarse_results==False:
+        return (multiscale_entropy)
 
-    return (multiscale_entropy)
+    elif return_coarse_results==True:
```

Comment:
I think returning the whole "mse" is probably the way to go for default.

Reply:
Let me know if you want me to include updating the default output in this pull request.

Reply:
I like the idea of returning both outputs, what are your thoughts about returning as 2 items instead of dict? Like this: 

`return (mse, multiscale_entropy)`

If there is a dict you would then need to know what the key is to access each part of the data, and that might be a pain/confusing.

Reply:
If that behaviour is good with you, I can update the output, remove the "option" for outputs, and add some info into the documentation about it. I'll leave the testing updates to you, and then we should be done?

Reply:
Yes, sounds perfect (maybe we could find other names than `mse`and `multiscale_entropy` which might look confusing)! Thanks for taking the time. Also, please add youself in the [contributors](https://github.com/neuropsychology/NeuroKit.py/blob/master/docs/about.rst) file ;)